<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCWindow</class>
 <widget class="QWidget" name="QCWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt200.qrc">
    <normaloff>:/resource/image/Title.png</normaloff>:/resource/image/Title.png</iconset>
  </property>
  <widget class="QLabel" name="lbItem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBatchNum">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>QC Batch Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQCSetting">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Qc Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQcTest">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxItem">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>16</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxBatchNum">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>16</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="CustomPlot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>761</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>360</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tips:QC tests only supports real-time test, please again in external reaction after the 
completion of the test</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qt200.qrc"/>
 </resources>
 <connections/>
</ui>
